### PR TITLE
Add tooltip for mig plan dropdowns

### PR DESCRIPTION
**Pull request by gildub**
plan-tooltip-edit

plan-tooltip-edit2

plan-tooltip-delete

Resolves https://github.com/konveyor/virt-ui/issues/223

### CODE REVIEW

**Comment by ibolton336** [comment]
@vconzola Does this text look good to you?

**Comment by ibolton336** [comment]
LGTM pending UX review @gildub

**Review by vconzola — CHANGES_REQUESTED** [review]
One tiny nit. We shouldn't use a contraction, "it's running". Should be "it is running". Otherwise, LGTM.

**Comment by mturley** [comment]
@gildub this also LGTM after adjusting the text per @vconzola's feedback

**Comment by vconzola** [comment]
@ibolton336 LGTM too.

File: src/app/Plans/components/PlanActionsDropdown.tsx [hunk 1]
```diff
@@ -30,28 +31,46 @@ const PlansActionsDropdown: React.FunctionComponent<IPlansActionDropdownProps> =
...
               hasCondition(conditions, PlanStatusAPIType.Executing) || deletePlanResult.isLoading
             }
-            onClick={() => {
-              setKebabIsOpen(false);
-              toggleDeleteModal();
-            }}
-            key="Delete"
+            content={
+              hasCondition(conditions, PlanStatusAPIType.Executing)
+                ? "This plan cannot be deleted because it's running"
```
Comment (mturley) [inline src/app/Plans/components/PlanActionsDropdown.tsx:56]: @gildub This is ready to merge as soon as we replace `it's` with `it is` in these two strings.

Comment (gildub) [reply src/app/Plans/components/PlanActionsDropdown.tsx]: Done

**Pull request merged**